NoSoft-SA/nspack
# - rename color to colour

Summary of this change:
renamed color_percentages to colour_percentages 

CHANGELOG entry:
Added
- colour_percentage to cartons, carton_labels, pallets and pallet_sequences grids
    
Changed
Renamed 
- color_applies to colour_applies on commodities table and CRUD
-  color_percentages to colour_percentages  table and CRUD
-  color_percentage_id to colour_percentage_id on product_setups, run_cache, carton_labels and pallet_sequences

How to test this code via the UI:

Masterfiles | Fruit | Commodities
Steps
    edit a commodity and set colour_applies to true
    add colour percentages to a commodity : colour percentages list
    edit a commodity and inline edit colour percentage (value should be between 1 and 100)

Production | Product Setups | Product Setups
    edit a product setup to add colour percentage (shown only if commodity.colour_applies)

Production | Runs | Production Runs
    allocate product setups and execute run
    colour_percentage_id will now be part of the run cache and pushed down to carton labels and pallet sequences on carton_labeling and palletizing respectively

Production | Reworks | Single Edit Pallet
    colour_percentage_id will now be part of the edit pallet sequence screen

Grids
Lists | Cartons
Lists | Carton Labels
List | Pallet Sequences

colour percentage and colour description will be displayed at the far right of the grid for Kromco
